uiCore/src/main/res/values/styles.xml

```diff
@@ -1,5 +1,114 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+
+    <style name="loginImageStyle">
+        <item name="android:layout_width">@dimen/iconSize</item>
+        <item name="android:layout_height">@dimen/iconSize</item>
+        <item name="android:src">@drawable/logo_render_full_notext</item>
+    </style>
+
+    <style name="titleStyle">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:ellipsize">end</item>
+        <item name="android:lines">1</item>
+        <item name="android:textAlignment">center</item>
+        <item name="android:textSize">@dimen/title2</item>
+        <item name="android:textColor">?legacyOnBackground</item>
+        <item name="android:padding">@dimen/gridx3</item>
+    </style>
+
+    <style name="errorTextStyle">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:textAlignment">center</item>
+        <item name="android:textColor">?legacyOnBackground</item>
+        <item name="android:textSize">@dimen/body1</item>
+        <item name="android:visibility">gone</item>
+        <item name="android:layout_marginHorizontal">@dimen/gridx5</item>
+        <item name="android:layout_marginVertical">@dimen/gridx3</item>
+    </style>
+
+    <style name="Theme.Forzenbook.colorBase" parent="Theme.MaterialComponents.DayNight.NoActionBar">
+        <item name="android:fontFamily">@font/mont_medium</item>
+        <item name="colorPrimary">@color/black</item>
+        <item name="fontFamily">@font/mont_medium</item>
+        <item name="legacyBackground">@color/lion_yellow</item>
+        <item name="legacyPrimary">@color/black</item>
+        <item name="legacyOnPrimary">@color/white</item>
+        <item name="legacySecondaryVariant">@color/white</item>
+        <item name="legacySurface">@color/dark_gray</item>
+        <item name="legacyOnSurface">@color/white</item>
+        <item name="legacyOnError">@color/red</item>
+        <item name="legacyOnBackground">@color/black</item>
+        <item name="android:statusBarColor">?attr/colorPrimaryVariant</item>
+    </style>
+
+    <style name="inputBodyStyle">
+        <item name="android:layout_width">match_parent</item>
```

Comment:
It is very rare for a style to include width and height. We can remove these so they are more flexible

Reply:
are they not already flexible because you can simply override the default values in styles? Removed, just curious.

Reply:
Width and height are special cases because you would need to come look at the defaults to know what the behavior would be. And overriding isn't always clean or reliable with them darn xmls. The coder should always expect to add at least a width and height